recipes/wrenfold/meta.yaml

```
@@ -0,0 +1,58 @@
+{% set version = "0.1.0" %}
+
+package:
+  name: wrenfold
+  version: {{ version }}
+
+source:
+  url: https://github.com/wrenfold/wrenfold/releases/download/v{{ version }}/wrenfold-source-v{{ version }}.tar.gz
+  sha256: c621dff6173822e165c112ea9ff491ecb3a5f16b01aedd890614fb1811e05a5a
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ stdlib("c") }}
+  host:
+    - python
+    - scikit-build-core ==0.9.8
+    - cmake >=3.20
+    - ninja >=1.5
+    - mypy ==1.9.0
+    - pip
+  run:
+    - python
+
+test:
+  imports:
+    - wrenfold
+    - wrenfold.sym
+  requires:
+    - pip
+    - numpy
+    - sympy
```

Why are they required here?
```suggestion
    - numpy
    - sympy
```

---

The command `test_wheel.py` runs the unit tests + examples, which depend on `numpy` and `sympy`.

They are a hard dependency only at test time. At runtime they are optional dependencies.